sycl/test-e2e/AddressSanitizer/nullpointer/nullpointer.cpp

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: linux
+// RUN-NOT: %{build} %device_asan_flags -O0 -g -o %t
+// RUN-NOT: %{run} not %t 2>&1 | FileCheck %s
```

Comment:
Why are those two RUN disabled? If we don't need them, can we remove them with a TODO?

Reply:
enabled